system/addon-manifest.xml

```diff
@@ -54,5 +54,6 @@
   <addon>xbmc.metadata</addon>
   <addon>xbmc.python</addon>
   <addon>xbmc.webinterface</addon>
+  <addon>service.autoexec</addon>
```

Comment:
I think it needs to be removed from here it is to be disabled by default if I understood @garbear correctly?

Reply:
Running autoexec.py, if present, should certainly be disabled by default. One option is to omit the add-on from the manifest. Instead, the approach taken in this PR is to ship an enabled add-on, with a setting that defaults to false.

Reply:
Ah, ok. Got it. Is there any benefit for this versus the other approach?

Reply:
Imo the add-on should be default disabled, and personally I'd take it further and not have it installed. Available in repo, user then chooses to install.